### PR TITLE
SJQ2T-66: Swagger documentation

### PR DESCRIPTION
**Pull request by MarkoRadunovicHtec**
Swagger documentation edited trough annotations, Names edited, Example request and return values added for every endpoint

### CODE REVIEW

File: backend/src/main/java/com/htecgroup/skynest/controller/UserController.java [hunk 1]
```diff
@@ -18,11 +26,83 @@
...
+                  schema = @Schema(implementation = ErrorMessage.class),
+                  examples = {
+                    @ExampleObject(
+                        value =
+                            "{\"messages\":[\"This email is already in use\"],"
+                                + " \"status\": \"400\","
+                                + " \"timestamp\": \"2022-06-03 16:18:12\"}")
+                  })
+            })
+      })
```
Comment (petarstoimenovskihtec) [inline backend/src/main/java/com/htecgroup/skynest/controller/UserController.java:95]: You don't expect error 500 here? :P :D  Applies to every endpoint

File: backend/src/main/java/com/htecgroup/skynest/controller/UserController.java [hunk 2]
```diff
@@ -18,11 +26,83 @@
...
+              schema = @Schema(implementation = UserRegisterRequest.class),
+              mediaType = "application/json",
+              examples = {
+                @ExampleObject(
+                    value =
+                        "{\"email\": \"username@gmail.com\","
+                            + " \"password\": \"paSword1\","
+                            + "  \"name\": \"Name\","
+                            + "  \"surname\": \"Surname\","
+                            + "  \"phoneNumber\": \"38166575757\","
```
Comment (petarstoimenovskihtec) [inline backend/src/main/java/com/htecgroup/skynest/controller/UserController.java:108]: Maybe add the phone format here if you have any ?

Comment (MarkoRadunovicHtec) [reply backend/src/main/java/com/htecgroup/skynest/controller/UserController.java:108]: When schema is opened in Request body, there is shown a pattern for phone number, which is only number. We didn't specify it any further

File: backend/src/main/java/com/htecgroup/skynest/controller/UserController.java (hunk 1 above)
Comment (MarkoRadunovicHtec) [reply backend/src/main/java/com/htecgroup/skynest/controller/UserController.java:95]: Definitely :D Also I saw in example that I missed status code.. Fixed it

Comment (petarstoimenovskihtec) [reply backend/src/main/java/com/htecgroup/skynest/controller/UserController.java:95]: maybe for consistency use the same format as the other error 500 desc

**Review by petarstoimenovskihtec — APPROVED** [review]

**Pull request merged**